GCD_of_n_numbers.cpp

```diff
@@ -1,23 +1,14 @@
-//This program aims at calculating the GCD of n numbers by division method
-#include <iostream>
-using namepsace std;
+#include <bits/stdc++.h>
+using namespace std;
 int main()
 {
-  cout <<"Enter value of n:"<<endl;
-  cin >> n;
+  int n;
```

Comment:
Please add a small user interface:
* What make the program?
* What want the program from the user?